src/main/scala/battleship/game/Game.scala

```diff
@@ -22,34 +22,34 @@ object Game {
 
```

Comment:
It seems to me that you only added the indentation in this file... If you want to remove this one from your pull request, see https://stackoverflow.com/a/44355701